v03_pipeline/lib/reference_datasets/gnomad_qc.py

```diff
@@ -0,0 +1,10 @@
+import hail as hl
+
+
+def get_ht(raw_dataset_path: str, *_) -> hl.Table:
+    if raw_dataset_path.endswith('mt'):
+        ht = hl.read_matrix_table(raw_dataset_path).rows()
+    else:
+        ht = hl.read_table(raw_dataset_path)
```

Comment:
This line shouldn't run right?  `get_ht` will only be called if the table doesn't exist?

Reply:
In the original config the 37 source is a matrix table and 38 is a hail table

```
    'gnomad_qc': {
        '37': {
            'version': 'v2',
            'custom_import': import_matrix_table,
            # Note: copied from 'gs://gnomad/sample_qc/mt/gnomad.joint.high_callrate_common_biallelic_snps.pruned.mt'
            'source_path': 'gs://seqr-reference-data/gnomad_qc/GRCh37/gnomad.joint.high_callrate_common_biallelic_snps.pruned.mt',
        },
        '38': {
            'version': '4.0',
            'path': 'gs://gcp-public-data--gnomad/release/4.0/pca/gnomad.v4.0.pca_loadings.ht',
        },
```

Reply:
ah, ok!

Reply:
yup that was a copy/paste typo, and I updated the gnomad_qc script to check `reference_genome` instead of the path suffix, which I agree is a better solution